scheduler/src/main/resources/applicationContext.xml

```diff
@@ -76,8 +80,30 @@
                 <prop key="org.quartz.jobStore.misfireThreshold">120000</prop>
                 <prop key="org.quartz.jobStore.driverDelegateClass">org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
                 </prop>
+                <prop key="org.quartz.jobStore.isClustered">true</prop>
+                <prop key="org.quartz.scheduler.instanceId">AUTO</prop>
             </props>
         </property>
     </bean>
     <bean id="restTemplate" class="org.springframework.web.client.RestTemplate"/>
+
+    <bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
```

Comment:
We tried using org.apache.http.client.HttpClient, and worked fine. Should we go ahead with it and then donot have to use the system properties.